docker/Dockerfile

```diff
@@ -117,6 +117,12 @@ RUN apt-get -qq update \
 
 ENV PATH=$PATH:/usr/lib/btbn-ffmpeg/bin
 
+# install go2rtc
+RUN wget -O go2rtc "https://github.com/AlexxIT/go2rtc/releases/download/v0.1-beta.9/go2rtc_linux_${TARGETARCH}" \
+    && chmod +x go2rtc \
+    && mkdir -p /usr/local/go2rtc/sbin/ \
```

Comment:
Shouldn't it be installed in `/usr/local/bin/go2rtc` instead? If you need a directory to store any contents of it like the final `go2rtc.yaml`, you should probably use `/usr/local/share/go2rtc/go2rtc.yaml`, or even `/etc/go2rtc.yaml` or `/etc/go2rtc/go2rtc.yaml`.

Reply:
I'm installing it in the same place that nginx is installed https://github.com/blakeblackshear/frigate/blob/dev/docker/rootfs/etc/services.d/nginx/run#L2